core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -502,7 +502,7 @@ protected Scope visitRefreshMaterializedView(RefreshMaterializedView refreshMate
             Query query = parseView(optionalView.get().getOriginalSql(), name, refreshMaterializedView);
             Scope queryScope = process(query, scope);
 
-            analysis.setUpdateType("INSERT", targetTable, Optional.empty());
+            analysis.setUpdateType("REFRESH MATERIALIZED VIEW", targetTable, Optional.empty());
```

Comment:
Few lines below we also do `checkCanInsertIntoTable` which I am not sure if it is ok here.